### PR TITLE
Added Ubuntu PPA for development version.

### PR DESCRIPTION
**Pull request by sergiobenrocha2**
The PPA is updated daily, automatically:

https://code.launchpad.net/~random-stuff/+recipe/reicast-daily

There's ARM package too, worked at least on odroid U3.

### CODE REVIEW

**Comment by skmp** [comment]
How do you get the ppa builds working? Is it possible to register reicast-team as a team there?

**Comment by sergiobenrocha2** [comment]
Sure. Do you have an account there? I can't find "reicast-team"

I imported the reicast code here: https://goo.gl/gZkfrL and there's a debian package branch here: https://goo.gl/XtAArI . The recipe build https://goo.gl/gco0VN nest the debian folder to the reicast source and there's an option to build daily or build on request. In the daily option, it'll build in the first import which has new commits (launchpad imports code every 5 hours), so if you do more commits later you'll have to wait for the next day (or just click in the "build now" or "Request build", to build again in the same day; the PPA has a limit of 5 build per day).

**Comment by sergiobenrocha2** [comment]
There's a "skmp" account there, are you using that?

**Comment by skmp** [comment]
Yes, that is me :p. Thanks for the invite!

On Fri, Aug 28, 2015 at 6:42 PM, sergio-br2 <notifications@github.com>
wrote:

> There's a "skmp" account there, are you using that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reicast/reicast-emulator/pull/819#issuecomment-135828715>
> .
>



-- 
~skmp

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Development/Beta versions
 | ![OSX](http://i.imgur.com/0YoI5Vm.png) OSX         | *TODO* | *TODO*
 
 
-Additional builds (iOS & android) can be found at [angelxwind's](http://reicast.angelxwind.net/) buildbot
+Additional builds (iOS & android) can be found at [angelxwind's](http://reicast.angelxwind.net/) buildbot and [Random Stuff "Daily/Nightly/Testing" PPA](https://launchpad.net/~random-stuff/+archive/ubuntu/ppa) (for Ubuntu).
```
Comment (Holzhaus) [inline README.md]: By the way, http://reicast.angelxwind.net/ looks down to me (DNS error).

**Comment by skmp** [comment]
@angelXwind ~ is reicast.angelxwind.net still valid and relevant? :)

**Pull request merged**